keyboards/cipulot/ec_virgo/rules.mk

```diff
@@ -0,0 +1,4 @@
+CUSTOM_MATRIX = lite
+ANALOG_DRIVER_REQUIRED = yes
+SRC += keyboards/cipulot/common/matrix.c keyboards/cipulot/common/ec_board.c keyboards/cipulot/common/ec_switch_matrix.c
```

Comment:
```suggestion
ANALOG_DRIVER_REQUIRED = yes
VPATH += keyboards/cipulot/common
SRC += matrix.c ec_board.c ec_switch_matrix.c
```

Reply:
Testing this. Once verified I'll apply. yhis and the locking items patch es to the remaining PRs open

Reply:
@waffle87 I tested and verified the patch. To save on unnecessary cloning of PRs I also applied the change for this include and the locking stuff to the other boards that are already merged.